.github/workflows/pr_title.yml

```diff
@@ -0,0 +1,13 @@
+name: Check PR title
+on:
+  pull_request:
+    types: [opened, edited]
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check PR title
+        run: |
+          title=$(jq -r .pull_request.title "$GITHUB_EVENT_PATH")
+          [[ $title =~ ^.*:\ .*$ ]] || exit 1
```

Comment:
Can we output an error message when the title is not qualified? Like "PR title should follow this format: XYZ"

Reply:
I'm a bit ambivalent about this.

Adding that would essentially double the amount of code (need print message then exit, so need a separate `if` or subprocess), and require maintaining the error message. Also I trust any hacker will immediately understand the issue. I'm happy to be convinced otherwise if you think it'll be a valuable UX improvement.

Reply:
Eh maybe you're right, it might be obvious what's the issue, but not necessarily obvious how to fix it. Let me update it.

Reply:
I am not sure whether everyone familiar with the regex. But an easy fix would be to update the github action stage name to "Check PR title to match format "*: *"